examples/image/cmap.py

```diff
@@ -0,0 +1,33 @@
+"""
+Simple Plot
+============
+Example showing simple plot creation and subsequent cmap change with 512 x 512 pre-saved random image.
+"""
+# test_example = true
+
+from fastplotlib import Plot
+import numpy as np
+from pathlib import Path
+
+from wgpu.gui.offscreen import WgpuCanvas
+from pygfx import WgpuRenderer
+
+canvas = WgpuCanvas()
+renderer = WgpuRenderer(canvas)
+
+plot = Plot(canvas=canvas, renderer=renderer)
+
+data_path = Path(__file__).parent.parent.joinpath("data", "random.npy")
+data = np.load(data_path)
```

Comment:
maybe let's choose a real image so that we can see if things are off when debugging etc. later. `imageio` has some grayscale test images too. Make sure the shape is actually `(x, y)` and not `(x, y, rgb)` when using it for examples with cmap https://imageio.readthedocs.io/en/v2.8.0/standardimages.html